R/package_installer.R

```diff
@@ -11,13 +27,17 @@
 #' }
 package_installer <- function() {
   pkgs_server <- get_server_path()
+
   pkg_name <- rstudioapi::showPrompt(
-    "Enter Package Name",
+    "Enter Package Name(s)",
     paste(
       "Install from",
       substr(pkgs_server, 6, nchar(pkgs_server))
     )
   )
```

Comment:
I think I'd expand this to explicitly say multiple packages can be installed by separating with a comma. I know that's probably instinct, but unless people spot the plural 'name(s)' they might not realise they can install multiple.

Reply:
Thanks I forgot to run `styler::style_pkg()` in my rush to do this before our meetings. Even though we were talking about it earlier :)